docs/user/installation.rst

```diff
@@ -121,6 +128,28 @@ PyPi
 
 `intelmqsetup` will create all necessary directories, provides a default configuration for new setups. See the :ref:`configuration` for more information on them and how to influence them.
 
+Docker
+^^^^^^
+
+Navigate to your preferred installation directory, i. e. use ``mkdir ~/intelmq && cd ~/intelmq``
+
+.. code-block:: bash
+
+   git clone https://github.com/certat/intelmq-docker.git
+
+   sudo docker pull certat/intelmq-full:1.0
+
+   mkdir intelmq_logs
+
+   cd intelmq-docker
+
+   chown -R $USER:$USER example_config
```

Comment:
If the user clones the repository, isn't the owner of the directory correctly set alread?

Reply:
I had some issues in the past, this is just in case :)